Error "type `Config` is not a member of trait `FromRequest`"

When I try to use "actix-web-grants" I get the following error:

error[E0437]: type `Config` is not a member of trait `FromRequest`
  --> C:\Users\myuser\.cargo\registry\src\github.com-1ecc6299db9ec823\actix-web-grants-3.0.0-beta.3\src\permissions\mod.rs:92:5

What could be the reason to get this error?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -10,7 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 
-## [Unreleased] - 2021-11-17
+## [v3.0.0-beta.4] - 2021-11-17
 ### Added
 - `actix-web: 4.0.0-beta.11` support [#22](https://github.com/DDtKey/actix-web-grants/pull/22)
 ### Changed
@@ -77,3 +77,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [v3.0.0-beta.1]: https://crates.io/crates/actix-web-grants/3.0.0-beta.1
 [v3.0.0-beta.2]: https://crates.io/crates/actix-web-grants/3.0.0-beta.2
 [v3.0.0-beta.3]: https://crates.io/crates/actix-web-grants/3.0.0-beta.3
+[v3.0.0-beta.4]: https://crates.io/crates/actix-web-grants/3.0.0-beta.4

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,6 +8,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 ### Changed
+
+
+## [Unreleased] - 2021-11-17
+### Added
+- `actix-web: 4.0.0-beta.11` support [#22](https://github.com/DDtKey/actix-web-grants/pull/22)
+### Changed
 - Remove unused `Config` type [#20](https://github.com/DDtKey/actix-web-grants/pull/20)
 
 ## [v3.0.0-beta.3] - 2021-10-11

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "actix-web-grants"
-version = "3.0.0-beta.3"
+version = "3.0.0-beta.4"
 authors = ["DDtKey <ddttkey@gmail.com>"]
 description = "Extension for `actix-web` to validate user permissions"
 repository = "https://github.com/DDtKey/actix-web-grants"
@@ -20,8 +20,8 @@ default = ["macro-check"]
 macro-check = ["actix-grants-proc-macro"]
 
 [dependencies]
-actix-web = { version = "4.0.0-beta.9", default-features = false }
-actix-grants-proc-macro = { path = "./proc-macro", version = "2.0.0-beta.3", optional = true }
+actix-web = { version = "4.0.0-beta.11", default-features = false }
+actix-grants-proc-macro = { path = "./proc-macro", version = "2.0.0-beta.4", optional = true }
 
 [dev-dependencies]
 actix-rt = "2"

--- a/examples/jwt-httpauth/Cargo.toml
+++ b/examples/jwt-httpauth/Cargo.toml
@@ -8,7 +8,7 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-actix-web = { version = "4.0.0-beta.9" }
+actix-web = { version = "4.0.0-beta.11" }
 actix-web-grants = { path = "../../" }
 actix-web-httpauth = "0.6.0-beta.2"
 jsonwebtoken = "7"

--- a/proc-macro/Cargo.toml
+++ b/proc-macro/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "actix-grants-proc-macro"
-version = "2.0.0-beta.3"
+version = "2.0.0-beta.4"
 authors = ["DDtKey <ddttkey@gmail.com>"]
 description = "A proc-macro way to validate user permissions for `actix-web-grants` crate."
 repository = "https://github.com/DDtKey/actix-web-grants"
@@ -19,5 +19,5 @@ proc-macro2 = "1.0"
 syn = { version = "1.0", features = ["full", "derive", "extra-traits"] }
 
 [dev-dependencies]
-actix-web = { version = "4.0.0-beta.9", default-features = false }
+actix-web = { version = "4.0.0-beta.11", default-features = false }
 actix-web-grants =  { path = "../" }

